src/android/build.gradle

```diff
@@ -46,7 +46,7 @@ android {
 }
 
 dependencies {
-    implementation('io.seon.androidsdk:androidsdk:4.1.0') {
+    implementation('io.seon.androidsdk:androidsdk:5.0.0') {
```

Comment:
Hi Shahin
Thanks for the update and great work.
As I can see and as this is a breaking change on [Android native seon library](https://github.com/seontechnologies/seon-android-sdk-public), the seon flutter plugin should be updated to v2.0.0.
<img width="752" alt="image" src="https://user-images.githubusercontent.com/22389242/212443522-0a64b412-b845-4af8-a92b-ccc9d6463d72.png">

I have tested on my real device and it sample project is working fine.
Please update below files with appropriate content:
src/CHANGELOG.md
src/pubspec.yaml

And please mention this is breaking change on Android based on seon native library on README files:
README.md
src/README

Reply:
Hello Farshid
Thanks for the review.
Your requested changes are committed.
Still haven't figured it out why the new version doesn't work on my real device since the previous versions are working fine, as this may lead to problem in production.

Reply:
Hi Shahin
Thanks for the changes.
So, I can not approve until you make sure it works on your device.
please investigate and find out why it is not working on your device.
Thanks

Reply:
hi @sefidgaran 
The issue is fixed https://github.com/seontechnologies/seon-android-sdk-public/issues/12
approve for merge please